hc/accounts/views.py

```diff
@@ -156,7 +156,7 @@ def profile(request):
         elif "update_reports_allowed" in request.POST:
             form = ReportSettingsForm(request.POST)
             if form.is_valid():
-                profile.reports_allowed = form.cleaned_data["reports_allowed"]
+                profile.reports_allowed = request.POST.get("reports_allowed", '0')
```

Comment:
Because our form data is requested via the `request.POST` method. and the `request.POST` and `request.GET` methods are what are used when accessing a normal request/response cycle.